core/src/test/java/tech/tablesaw/table/TableSummaryTest.java

```diff
@@ -0,0 +1,66 @@
+package tech.tablesaw.table;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+import tech.tablesaw.api.DoubleColumn;
+import tech.tablesaw.api.StringColumn;
+import tech.tablesaw.api.Table;
+
+public class TableSummaryTest {
+  @Test
+  public void summaryDoublesOnly() {
+    DoubleColumn value = DoubleColumn.create("value1").append(1.0).append(1.1).append(1.2);
+    DoubleColumn value2 = DoubleColumn.create("value2").append(2.0).append(2.1).append(2.2);
+
+    Table testTable = Table.create("Data");
+    testTable.addColumns(value);
+    testTable.addColumns(value2);
+    Table result = testTable.summary();
+    assertEquals(
+        "                            Data                             \n"
+            + " Summary   |         value1         |        value2         |\n"
+            + "-------------------------------------------------------------\n"
+            + "    Count  |                     3  |                    3  |\n"
+            + "      sum  |                   3.3  |                  6.3  |\n"
+            + "     Mean  |                   1.1  |                  2.1  |\n"
+            + "      Min  |                     1  |                    2  |\n"
+            + "      Max  |                   1.2  |                  2.2  |\n"
+            + "    Range  |   0.19999999999999996  |  0.20000000000000018  |\n"
+            + " Variance  |  0.009999999999999995  |  0.01000000000000004  |\n"
+            + " Std. Dev  |   0.09999999999999998  |   0.1000000000000002  |",
+        result.print());
+  }
+
+  @Test
+  public void summaryMixedTypes() {
+    StringColumn label =
+        StringColumn.create("label").append("yellow").append("green").append("blue");
+    DoubleColumn value = DoubleColumn.create("value1").append(1.0).append(1.1).append(1.2);
+    StringColumn label2 = StringColumn.create("label2").append("red").append("red").append("red");
```

Comment:
Maybe instead of having a second string and double column we could show some other types like `BooleanColumn` and `DateColumn`?